### ISSUE TITLE
Tag images to reflect alt text or no alt text

### ISSUE THREAD

**Issue by anneke** [elk-zone/elk#1035]
### Clear and concise description of the problem

As a user, I want to see if an image has a description in the form of an alt text, so I can make a conscious choice to boost it or not 

### Suggested solution

On front-end social, the mods have achieved this by adding the following css:

```
/* show when images have alt text */
.media-gallery__item {
  --alt-crop-x: 6ch;
  --alt-crop-y: 2em;
  background: darkgreen;
  color: white;
}

/* only revealed by cutaway when alt is defined */
.media-gallery__item::after {
  content: "alt";
  display: grid;
  inset-block-end: 0;
  inset-inline-start: 0;
  min-height: var(--alt-crop-y);
  min-width: var(--alt-crop-x);
  place-content: center;
  position: absolute;
}

/* cut away to reveal badge */
.media-gallery__item img[alt] {
  clip-path: polygon(0 0, 100% 0, 100% 100%, var(--alt-crop-x) 100%, var(--alt-crop-x) calc(100% - var(--alt-crop-y)), 0 calc(100% - var(--alt-crop-y)));
}

/* Remove the clip path if we can do better with :has() */
.media-gallery__item:has(img) img[alt] {
  clip-path: none;
}

/* Apply better alt-text badge over top of image */
.media-gallery__item:has(img)::after {
  --alt-crop-x: initial;
  --alt-crop-y: initial;
  background: darkgreen;
  border: thin solid rgba(255, 255, 255, 0.4);
  border-radius: 3px;
  inset-block-end: 0.5em;
  inset-inline-start: 0.5em;
  padding: 0.25rem 0.5rem;
  position: absolute;
  pointer-events: none;
  z-index: 2;
}

/* Change badge when no alt is defined */
.media-gallery__item:has(img:not([alt]))::after {
  background: maroon;
  content: "no alt";
}
```

### Alternative

I think the css-only solution is very elegant :) I think it was coded by either Miriam Suzanne or Estelle Weyl, props go to them (or another mod of the front-end.social server that was involved!)

**Comment by michelc** [comment]
Bonsoir @anneke 

I think it's already done?

When image image has a description there is a "ALT" mention in the bottom left: https://mastodon.top/@LeMotLeJeu/109602239299992824

And when there is no description, the "ALT" mention is not there: https://mastodon.top/@LeMotLeJeu/109597068145681981

**Comment by ayoayco** [comment]
Closing this as we now "Tag" images clearly if they have alt texts
